compiler/passes/docs.cpp

```diff
@@ -294,8 +295,9 @@ void generateSphinxOutput(std::string sphinxDir, std::string outputDir) {
   const char * venvBinDir = astr(venvDir, "/bin");
   const char * sphinxBuild = astr(venvBinDir, "/sphinx-build");
 
-  const char * envVars = astr("export PATH=", venvBinDir, ":$PATH && ",
-                              "export VIRTUAL_ENV=", venvDir);
+  const char * envVars = astr("export PATH=", venvBinDir, ":$PATH && \
+                              export VIRTUAL_ENV=", venvDir, " && \
+                              export CHPLDOC_AUTHOR='", fDocsAuthor, "'");
 
```

Comment:
I had forgotten this approach to splitting literal strings across multiple lines.  I am fine with this but wondered if you knew that the C compiler will automatically concatenate adjacent literal strings?